adr/ADR-11.md

```diff
@@ -0,0 +1,92 @@
+# JetStream API unsigned 64 bit numerical value fields
+
+|Metadata|Value|
+|--------|-----|
+|Date    |2021-07-15|
+|Author  |@scottf|
+|Status  |Approved|
+|Tags    |server, client, jetstream|
+
+## Context
+
+This document summarizes fields in the JetStream api schema that are unsigned 64 bit numerical values. 
+The most up-to-date information can be found in the [schema](https://github.com/nats-io/jsm.go/tree/main/schemas/jetstream/api/v1)
+
+## Schema Object Fields
+
+Object and list of fields within the object that are unsigned 64 bit values.
+
+#### `consumer_configuration`
+ 
+* `opt_start_seq`
+
+#### `consumer_create_request`
+
+* `opt_start_seq`
+
+#### `consumer_create_response`
+
+* `consumer_seq`
+* `stream_seq`
+
+#### `consumer_info_response`
+
+* `opt_start_seq`
```

Comment:
Anything in consumer_config + consumer/stream which is a sequence pair for Delivered and AckFloor.
But missing here is `NumPending     uint64          `json:"num_pending"`